.github/workflows/lint_python.yml

```diff
@@ -14,16 +14,9 @@ jobs:
       - run: codespell --skip="*.tap"  # --ignore-words-list="" 
```

Comment:
Please change line 12 to `- run: bandit --recursive --skip=B101,B604,B607 .` to placate `bandit`.